### DISCUSSION TITLE
how to write varchar field for hybrid seach expr

### DISCUSSION THREAD

**Discussion by OldRiverChen** [milvus-io/milvus#25007] [Q&A and General discussion] [answered]
I wrote expr : "model == chatgpt && sceneId == 1111 && speaker == river"，but unwork

the error msg is "cannot parse expression: model == chatgpt && sceneId == 1111 && speaker == river, error: comparisons between string and non-string are not supported"

**Comment by yhmo** [comment, marked as answer]
"model == \"chatgpt\" && sceneId == 1111 && speaker == \"river\""
